itest/helper/helper.go

```diff
@@ -245,13 +245,11 @@ func (b *DeploymentBuilder) Build() *appsv1.Deployment {
 	}
 }
 
-var (
-	trivyScanner = v1alpha1.Scanner{
-		Name:    "Trivy",
-		Vendor:  "Aqua Security",
-		Version: "0.16.0",
-	}
-)
+var trivyScanner = v1alpha1.Scanner{
```

Comment:
No sure we want to be so strict about linting Go code. At some point we are probably going to spend most of the time making linter happy, whereas such things are not very relevant and super critical. I'd just enable linters that are supported by popular IDEs and check for real problems.